docs/2.0/installation.md

```diff
@@ -8,10 +8,34 @@ description: Instructions on how to install the league/commonmark library
 
 The recommended installation method is via Composer.
 
-In your project root just run:
-
 ```bash
-composer require league/commonmark:^2.0
+composer require "league/commonmark:^2.0"
+```
+
+Since this version is still a work in progress, you will need to include packages of dev stability. If you see an error like this you need to change your stability requirements.
+
+> Your requirements could not be resolved to an installable set of packages.
+>  
+> Problem 1
+> - The requested package league/commonmark ^2.0 is satisfiable by league/commonmark[2.0.x-dev] but these conflict with your requirements or minimum-stability.
+
+The easiest way to do this, is to create a file in your project root, called `composer.json`
+
+Put the following code in it and save:
+
+```json
+{
+	"require-dev": {
+		"league/commonmark": "^2.0@dev",
+		"dflydev/dot-access-data": "^3.0@dev"
+	}
+}
+```
+
+When the file is set up, run the following command and you are done.
+
+```sh
```

Comment:
I've been doing some consistency reviews of `docs/` and `bash` is preferred over `sh` for code block language selection.

```suggestion
```bash
```

Reply:
Yep, saw your work in #572 , didn't mean to undo that. Thanks for noticing/commenting, fixed it! :)